homeassistant/components/zha/number.py

```diff
@@ -971,6 +972,28 @@ class ThermostatLocalTempCalibration(ZHANumberConfigurationEntity):
     _attr_icon: str = ICONS[0]
 
 
+@MULTI_MATCH(
```

Comment:
You should also be able to use `CONFIG_DIAGNOSTIC_MATCH` and remove the `MULTI_MATCH` part at the top.
`CONFIG_DIAGNOSTIC_MATCH` is also based on `MULTI_MATCH` but has some small differences regarding cluster claiming IIRC

Reply:
I only used it because @dmulcahey suggested it. What is actually the difference? Are the args just interchangeable?

Reply:
> Are the args just interchangeable?

Yes, it should basically be the same. `MULTI_MATCH` is used for "normal" entities and the config/diagnostic match for those.

> I only used it because @dmulcahey suggested it.

I think he was rather linking the example for the `stop_on_match_group` part.

Reply:
So, I used `CONFIG_DIAGNOSTIC_MATCH` now, but it somehow doesn't work anymore. I also inherited from `ThermostatLocalTempCalibration` instead of `ZHANumberConfigurationEntity`. Any idea why? For me everything looks correct.

Reply:
Add `stop_on_match_group` to both classes.
(If it still doesn't work then, maybe try just moving the Sonoff TRV above the other one)

Reply:
This seems to work.